src/main/kotlin/com/sourcegraph/cody/statusbar/ReportCodyBugAction.kt

```diff
@@ -2,9 +2,9 @@ package com.sourcegraph.cody.statusbar
 
 import com.intellij.ide.BrowserUtil
 import com.intellij.openapi.actionSystem.AnActionEvent
-import com.intellij.openapi.project.DumbAwareAction
+import com.sourcegraph.common.ui.DumbAwareBGTAction
 
-class ReportCodyBugAction : DumbAwareAction("Open GitHub To Report Cody Issue") {
+class ReportCodyBugAction : DumbAwareBGTAction("Open GitHub To Report Cody Issue") {
   override fun actionPerformed(p0: AnActionEvent) {
     BrowserUtil.open("https://github.com/sourcegraph/jetbrains/issues/new?template=bug_report.yml")
```

Comment:
where is it used? this action? don't we need to put the label param btw?

Reply:
In the `CodyStatusBarActionGroup`.
What label param?